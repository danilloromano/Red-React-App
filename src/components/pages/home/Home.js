import React, { Component, Fragment } from "react";
import "../Home/Home.scss";

class Home extends Component {
    render() {
        return (
            <Fragment>

            </Fragment>
        )
    }
}

export default Home;