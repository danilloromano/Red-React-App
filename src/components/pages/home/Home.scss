@import '../../../assets/variables/variables.scss';

.container {
    flex: 1;

    &__form {
        display: flex;
        flex-direction: column;
        align-items: center;
        padding: 40px;
    }

    &__card {
        position: absolute;
        left: 50%;                   
        top: 30%;   
        transform: translateX(-50%); 
    }
}

@media screen and (min-width: 1024px) {

    .container {

        &__form {
            display: flex;
            flex: 1;
            flex-direction: column;
            justify-content: center;
            padding: 0px 50px 50px 140px;
        }

        &__card {
            position: absolute;
            top: 35%;
            left: 18%;                   
        }
    }
}