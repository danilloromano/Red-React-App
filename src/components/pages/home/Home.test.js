import React from 'react';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home render test', () => {

    it('Home should render correctly', () => {
        const wrapper = shallow(<Home />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });

});