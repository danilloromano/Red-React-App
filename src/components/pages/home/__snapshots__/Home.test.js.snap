// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render test Home should render correctly 1`] = `
<Fragment>
  <SideBarHeader />
  <div
    className="container__card"
  >
    <Connect(Card) />
  </div>
  <div
    className="container__form"
  >
    <StepperDesk />
    <Connect(Form) />
  </div>
</Fragment>
`;
